data-processing/scripts/fetch_arxiv_sources.py

```diff
@@ -0,0 +1,46 @@
+import argparse
+import os
+
+from common import directories
+from common.fetch_arxiv import fetch_from_arxiv
+from common.unpack import unpack, unpack_archive
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
+        "Fetch and unpack sources for a single arXiv paper."
+    )
+    parser.add_argument(
+        "arxiv_id",
+        help="The arXiv ID for a paper. May include version number (i.e., 'v1', 'v2', etc.)",
+    )
+    parser.add_argument(
+        "--output-dir",
+        help="Directory into which the arXiv sources will be fetched.",
```

Comment:
consider noting the sources will be stored in a  arxiv id subfolder of this location (so no need to make your own outputdir the arxiv id)

Reply:
Good thinking! Will do 👍